notificationsManager/Operators/UpdateOperatorsService.php

```diff
@@ -5,22 +5,27 @@
 use Exception;
 use NotificationsManager\Operators\Database\Entities\Operator;
 use NotificationsManager\Operators\Repositories\OperatorRepository;
+use NotificationsManager\Operators\Repositories\OperatorsRequestRepository;
 
-class UpdateOperatorsService
+readonly class UpdateOperatorsService
 {
-    public function __construct(private readonly OperatorRepository $operatorRepository)
-    {
+    public function __construct(
+        private OperatorRepository $operatorRepository,
+        private OperatorsRequestRepository $operatorsRequestRepository
+    ) {
     }
 
     /**
-     * @param array<int> $operatorData
      * @throws Exception
      */
-    public function update(array $operatorData): void
+    public function update(): void
     {
+        $operatorsData = $this->operatorsRequestRepository->getOperators();
         try {
-            $operator = new Operator($operatorData);
-            $this->operatorRepository->save($operator);
+            foreach ($operatorsData as $operatorData) {
+                $operator = new Operator($operatorData);
```

Comment:
Que te parece el tener que crear un Operador en cada data que tienes?? Pasar esta responsabilidad a otro lado??

Fíjate aquí:
$operatorsData = $this->operatorsRequestRepository->getOperators();

Que te parece si dentro de esa función mappeas los operadores y te devuelves un array de Operators? y aquí así solo tendrías que hacer el save